Scouting/src/main/java/ca/team2706/scouting/mcmergemanager/backend/dataObjects/CommentList.java

```diff
@@ -0,0 +1,108 @@
+package ca.team2706.scouting.mcmergemanager.backend.dataObjects;
+
+import android.util.Log;
+import android.widget.Toast;
+
+import ca.team2706.scouting.mcmergemanager.steamworks2017.dataObjects.Comment;
+import 	org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by Merge on 2017-12-06.
+ */
+
+public class CommentList {
+
+    /*Example JSON file
+    "{
+    "name":"Merge Robotics",
+    "number":2706,
+    "opr":0,
+    "dpr":0,
+    "comments":[
+        {
+            "body":"Test text",
+        },
+        {
+            "body":"Test text2",
+        }
+    ],
+    "pictures":[
+        {
+            "picture_type":"test",
+            "taken_at":null,
+            "link":"http://beancode.org/eastereggs/help.php"
+        }
+    ]
+}"
+    */
+
+    private ArrayList<String> comments = new ArrayList<>();
+    private int teamNumber;
+
+    public static final String JSONKEY_TEAMNO = "number";
+    public static final String JSONKEY_COMMENT = "comments";
+
+    /** Constructor
+     *  Create new comment list for @param teamNumber
+     */
+
+    public void CommentList(int teamNumber) {
```

Comment:
A constructor needs to be "public CommentList(..)", you don't put a return type on constructors (because it automatically returns a copy of this object).